code/modules/virus2/effect/stage_3.dm

```diff
@@ -178,6 +178,7 @@
 	name = "Pierrot's Throat"
 	desc = "Overinduces a sense of humor in the infected, causing them to be overcome by the spirit of a clown."
 	stage = 3
+	max_multiplier = 4
 	badness = EFFECT_DANGER_HINDRANCE
 
 /datum/disease2/effect/pthroat/activate(var/mob/living/mob)
```

Comment:
oh okay it wasn't your fault they had been using bad arg names already

still, gross